packages/server/src/authentication/verifyAuthenticationResponse.ts

```diff
@@ -247,6 +251,8 @@ export type VerifiedAuthenticationResponse = {
     userVerified: boolean;
     credentialDeviceType: CredentialDeviceType;
     credentialBackedUp: boolean;
+    origin: string;
+    rpId: string;
```

Comment:
I'm sure it's driven people nuts that I do this, but in this project the "D" in the camel-cased "RP ID" is always capitalized:

```suggestion
    rpID: string;
```

Reply:
your code is very heterogeneous about this naming, i have checked other interfaces and you use more rpId than rpID

Reply:
...Shoot, really? Am I at least consistent in the public API of the core methods? 😅 

I'll take a look when I'm on a computer, I thought I was better about using "`rpID`"...

Reply:
well I see:

- expectedRPID
- rpIdHash
- matchExpectedRPID
- rpID

but in typescript-types it's `PublicKeyCredentialRequestOptionsJSON.rpId`

let's rename then

Reply:
In the **public** API for @simplewebauthn/server methods I think I've been consistent with casing on inputs so far:

- `generateRegistrationOptions()`: `rpID`
- `verifyRegistrationResponse()`: `expectedRPID`

- `generateAuthenticationOptions()`: `rpID`
- `verifyAuthenticationResponse()`: `expectedRPID`

The point I was attempting to make is that I think it would be weird to have "`rpID`" as an input, and then get "`rpId"` out. But you've already agreed with me and renamed to `rpID` so I suppose this'll serve as something I can point to in the future whenever someone attempts to introduce `rpId` into the library's public API 😛 

> but in typescript-types it's `PublicKeyCredentialRequestOptionsJSON.rpId`

That's why I supposed that I've probably driven people nuts by going with `rpID` in this package while in the WebAuthn spec they use `rpId`. `PublicKeyCredentialRequestOptionsJSON.rpId` maps to the data structure in the L3 Draft of the spec:

https://w3c.github.io/webauthn/#dictdef-publickeycredentialrequestoptionsjson

In typical use of SimpleWebAuthn, pairing **@simplewebauthn/browser** with **@simplewebauthn/server**, the nice thing is that an RP doesn't need to think about things like this!